geotrek/api/v2/serializers.py

```diff
@@ -788,6 +767,28 @@ def get_departure_city(self, obj):
             city = zoning_models.City.objects.all().filter(geom__contains=geom).first()
             return city.code if city else None
 
+        def _replace_image_paths_with_urls(self, data):
+            def replace(html_content):
+                if not html_content:
+                    return html_content
+                soup = BeautifulSoup(html_content, features="lxml")
+                imgs = soup.find_all('img')
+                for img in imgs:
+                    if img.attrs['src'][0] == '/':
+                        base_url = self.context.get("request").build_absolute_uri("/")[:-1]
```

Comment:
try to use directly build_absolute_url to get final url : self.context.get("request").build_absolute_uri(img.attrs['src'])

Reply:
Right, I have done it.